src/test/java/api/requests/ItemsLostRequiringActualCostsTests.java

```diff
@@ -0,0 +1,243 @@
+package api.requests;
+
+import static api.support.JsonCollectionAssistant.getRecordById;
+import static org.folio.circulation.domain.policy.Period.minutes;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.joda.time.DateTime.now;
+import static org.joda.time.DateTimeZone.UTC;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Optional;
+
+import org.folio.circulation.support.http.client.Response;
+import org.hamcrest.CoreMatchers;
+import org.joda.time.DateTime;
+import org.junit.Test;
+
+import api.support.builders.CheckOutByBarcodeRequestBuilder;
+import api.support.builders.DeclareItemLostRequestBuilder;
+import api.support.builders.ItemBuilder;
+import api.support.builders.LostItemFeePolicyBuilder;
+import api.support.http.IndividualResource;
+import api.support.http.ItemResource;
+import api.support.http.ResourceClient;
+import api.support.matchers.ItemMatchers;
+import api.support.spring.SpringApiTest;
+import io.vertx.core.json.JsonObject;
+
+public class ItemsLostRequiringActualCostsTests extends SpringApiTest {
+  public ItemsLostRequiringActualCostsTests() {
+    super(true, true);
+  }
```

Comment:
Do we need to interact with loan history? If don't then we can safely remove the constructor.

Reply:
When the loan history is false, the tests were failing.
But that was many iterations ago.
I will check to see if I can get away with removing that or not.